packages/office-addin-debugging/src/start.ts

```diff
@@ -58,6 +62,14 @@ export function parseAppType(text: string): AppType | undefined {
             return AppType.Desktop;
         case "web":
             return AppType.Web;
+        case "win32":
```

Comment:
This may be OK, or might be better as a platform enum. I was thinking that appType and platform are two separate values.